packages/airnode-validator/src/config/config.test.ts

```diff
@@ -69,3 +69,57 @@ describe('nodeSettingsSchema', () => {
     );
   });
 });
+
+it('fails if ois.apiSpecifications.security.<securitySchemeName> is defined in ois.apiSpecifications.components.<securitySchemeName> but apiCredentials.securitySchemeValue is empty string', () => {
+  const config: Config = JSON.parse(
+    readFileSync(join(__dirname, '../../test/fixtures/interpolated-config.valid.json')).toString()
+  );
+
+  const securitySchemeName = 'Currency Converter Security Scheme';
+  const securityScheme = {
+    [securitySchemeName]: {
+      in: 'query',
+      type: 'apiKey',
+      name: 'access_key',
+    },
+  };
+  const apiCredentials = [
```

Comment:
Why not remove this and pass empty `apiCredentials` to the config? The config setup would be shorter.

Reply:
No reason. I just thought it might be closer to a real use-case if there was some data in apiCredentials.

Reply:
You can remove this then and rename `configWithSecuritySchemes` to `invalidConfig`.